router-dev/Chart.yaml

```diff
@@ -10,8 +10,8 @@ details: |-
   The Deis Router is a simple program that manages Nginx and Nginx configuration for managing
   ingress to web applications in a Kubernetes cluster without requiring each to have their own
   external (to the cluster) load balancer.  Router(s) will require a single external load balancer,
-  handle all ingress, and direct all incoming traffic to appropriate services within the cluster.  
-  
+  handle all ingress, and direct all incoming traffic to appropriate services within the cluster.
```

Comment:
I'll leave this for a different PR, if that's okay.  fwiw, this was an auto-whitespace change from my atom editor that I can remove if preferred...